docs/api_core.rst

```diff
@@ -1926,6 +1926,14 @@ The following annotations can be specified using the variable-length
    mixed enum types (such as ``Shape.Circle + Color.Red``) are
    permissible.
 
+   .. cpp:struct:: is_flag_enum
```

Comment:
Shall we give it a shorter name? `is_flag`? (for symmetry with the arithmetic flag, which is called `is_arithmetic`)